src/todo.js

```diff
@@ -51,51 +44,67 @@ export default class Todo {
         </div>
        
         `;
+  });
+
+  const deleteBtn = document.querySelectorAll('#delete');
+  const textLabel = document.querySelectorAll('#description');
+  const dropdownBtn = document.querySelectorAll('#dropdown');
+  const divDropdownBtn = document.querySelectorAll('#sectiontohide');
+  const checkBoxes = document.querySelectorAll('#todoCheck');
+
+  // eslint-disable-next-line
+  for (let index = 0; index < checkBoxes.length; index++) {
+    if (checkBoxes[index].dataset.todocompleted === 'true') {
+      checkBoxes[index].checked = true;
+    }
+  }
 
-      if (todo.completed) {
-        document.getElementById('todoCheck').checked = true;
-      }
-    });
-
-    const deleteBtn = document.querySelectorAll('#delete');
-    const textLabel = document.querySelectorAll('#description');
-    const dropdownBtn = document.querySelectorAll('#dropdown');
-    const divDropdownBtn = document.querySelectorAll('#sectiontohide');
-
-    deleteBtn.forEach((button) => {
-      button.addEventListener('click', (event) => {
-        const id = parseInt(event.target.getAttribute('data-index'), 10);
-        let localStoragetodos = this.getAllTodos();
-        localStoragetodos = localStoragetodos.filter((todo) => todo.index !== id);
-        // eslint-disable-next-line
+  deleteBtn.forEach((button) => {
+    button.addEventListener('click', (event) => {
+      const id = parseInt(event.target.getAttribute('data-index'), 10);
+      let localStoragetodos = getAllTodos();
+      localStoragetodos = localStoragetodos.filter((todo) => todo.index !== id);
+      // eslint-disable-next-line
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.